tweepy/streaming.py

```diff
@@ -964,6 +964,6 @@ class StreamRule(NamedTuple):
     .. _operators: https://developer.twitter.com/en/docs/twitter-api/tweets/search/integrate/build-a-query
     .. _Academic Research Project: https://developer.twitter.com/en/docs/projects
     """
-    value: str = None
-    tag: str = None
-    id: str = None
+    value: Optional[str] = None
+    tag: Optional[str] = None
+    id: Optional[str] = None
```

Comment:
`str | None` can be used with `from __future__ import annotations`.

Reply:
ed58b61